app/lib/bk/compat/parsers/harness/steps/run.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_relative '../../../pipeline/step'
-
```

Comment:
This file also uses the command step class so it should include the dependencies explicitly:

```suggestion
require_relative '../../models/steps/command'

```

Reply:
Adjusted 👍